csharp/src/Google.Protobuf.Test/CodedInputStreamExtensions.cs

```diff
@@ -49,5 +49,15 @@ public static T ReadMessage<T>(this CodedInputStream stream, MessageParser<T> pa
             stream.ReadMessage(message);
             return message;
         }
+
+#if !PROTOBUF_NO_ASYNC
+        public static T ReadMessage<T>(this CodedInputStream stream, AsyncMessageParser<T> parser)
```

Comment:
This is no longer necessary due to our change in the class hierarchy. Because now AsyncMessageParser<T> inherits MessageParser<T> and the original is valid.. I will remove this.